sidekick_core/lib/src/commands/plugins/install_plugin_command.dart

```diff
@@ -76,12 +76,12 @@ class InstallPluginCommand extends Command {
         case 'path':
           final dir = Directory(packageNameOrGitUrlOrLocalPath);
           if (!dir.existsSync()) {
-            throw "Directory at ${dir.absolute.path} does not exist";
+            error("Directory at ${dir.absolute.path} does not exist");
           }
 
           final localPackage = DartPackage.fromDirectory(dir);
           if (localPackage == null) {
-            throw "Directory at ${dir.absolute.path} is not a dart package";
+            error("Directory at ${dir.absolute.path} is not a dart package");
```

Comment:
We shouldn't use `error` because it calls `exit` which makes testing impossible

Reply:
Ah right, I forgot about that